Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -35,10 +38,10 @@
 INCIDENTS_PER_FETCH = int(demisto.params().get('incidents_per_fetch', 15))
 # Remove proxy if not set to true in params
 if not demisto.params().get('proxy'):
-    del os.environ['HTTP_PROXY']
```

Comment:
why did you change that ?

Reply:
I chaged it when it works without the keys

Reply:
not sure i understand..

Reply:
`del os.environ['HTTP_PROXY']` will make `KeyError` when HTTP_PROXY doesn't exists in os.environ. `os.environ('HTTP_PROXY', None)` works in that case.

Reply:
better to call `handle_proxy()`